<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editar_usuario</class>
 <widget class="QDialog" name="editar_usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Usuário</string>
  </property>
  <widget class="QTableView" name="login_tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="pesquisar_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Pesquisar Usuário</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>721</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>8</number>
   </property>
   <widget class="QWidget" name="Vendas">
    <attribute name="title">
     <string>Vendas</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>175</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="vendas_vende_checkBox">
        <property name="text">
         <string>Vendas &gt;&gt; Vender</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Compras">
    <attribute name="title">
     <string>Compras</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>222</width>
       <height>164</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="compras_orcamentos_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Orçamentos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_pedidos_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Pedidos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_fornecedores_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Fornecedores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_devolucoes_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Devoluções</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_centraldecompras_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Central de Compras</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compras_estoque_checkBox">
        <property name="text">
         <string>Compras &gt;&gt; Estoque</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Cadastro">
    <attribute name="title">
     <string>Cadastro</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>175</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cadastrar_cliente_checkBox">
        <property name="text">
         <string>Cadastrar Cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cadastrar_funcionario_checkBox">
        <property name="text">
         <string>Cadastrar Funcionário</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cadastrar_fornecedor_checkBox">
        <property name="text">
         <string>Cadastrar Fornecedor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Cliente">
    <attribute name="title">
     <string>Cliente</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>175</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="localizar_cliente_checkBox">
        <property name="text">
         <string>Localizar Cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="editar_cliente_checkBox">
        <property name="text">
         <string>Editar cliente</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Estoque">
    <attribute name="title">
     <string>Estoque</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Financeiro</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Relatorio">
    <attribute name="title">
     <string>Relatório</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Modulo">
    <attribute name="title">
     <string>Módulo</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Ferramentas">
    <attribute name="title">
     <string>Ferramentas</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>234</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QCheckBox" name="cadastrar_usuario_checkBox">
        <property name="text">
         <string>Cadastrar Usuário</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="editar_usuario_checkBox">
        <property name="text">
         <string>Editar Usuário</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="salvar_pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar Alterações</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="senha_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Alterar Senha</string>
   </property>
  </widget>
  <widget class="QLabel" name="permissoes_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Permissões</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="administrador_checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Definir usuário como administrador</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="deletar_usuario_checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Deletar Usuário</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
